src/Sulu/Bundle/AdminBundle/Resources/js/containers/CKEditor5/README.md

```diff
@@ -22,7 +22,7 @@ const handleBlur = () => alert('Text editing finished!');
 The editor can be extended by adding more plugins and more configuration. That's what the `PluginRegistry` and
 `ConfigRegistry` are for:
 
-```javascript
+```javascript static
```

Comment:
looks misplaced

Reply:
It actually is not. This makes the code below not rendered in the styleguide. Otherwise we would have to add the ckeditor5-font package somehow, because we get errors.

Reply:
ok good to know